jvb/src/main/java/org/jitsi/videobridge/Videobridge.java

```diff
@@ -949,14 +949,20 @@ public IQ healthCheckIqReceived(@NotNull HealthCheckIQ iq)
          * channels failed because there was no transport activity (which
          * includes those that failed because there was no payload activity).
          */
-        public AtomicInteger totalFailedConferences = new AtomicInteger(0);
+        public CounterMetric totalFailedConferences = VideobridgeMetricsContainer.getInstance().registerCounter(
+                "failed_conferences",
+                "The total number of conferences that had ALL of their channels failed because " +
```

Comment:
Can you please change the descriptions to something like "Number of conferences in which ALL/SOME of the endpoints failed ICE"? It'd be nice to update/clarify the descriptions while porting and this one is outdated. I actually question the usefulness of these 2 metrics altogether.

Reply:
Sure thing!